digital_asset_types/src/dao/mod.rs

```diff
@@ -245,6 +249,26 @@ impl SearchAssetsQuery {
             joins.push(rel);
         }
 
+        if let Some(n) = self.name.to_owned() {
+            let name_as_str = std::str::from_utf8(&n).map_err(|_| {
+                DbErr::Custom(
+                    "Could not convert raw name bytes into string for comparison".to_owned(),
+                )
+            })?;
+
+            let cond = Condition::all().add(asset_data::Column::RawName.like(name_as_str));
```

Comment:
```suggestion
            let name_expr =
                SimpleExpr::Custom(format!("chain_data->>'name' = '{}'", name_as_str).into());
            conditions = conditions.add(name_expr);
```

Reply:
this would force a full match, right? could this be adjusted to be partial matches, similar to what a `like` expression would do?

Reply:
Sure, I can change it to make a partial match

Reply:
what is the updated suggestion i need to change in the PR before merging?